src/consistency-testing/chaostest/chaostest/kafka_cluster.py

```diff
@@ -267,6 +280,10 @@ def __init__(self, config):
             config_node["id"]: KafkaNode(config, config_node["id"])
             for config_node in config["nodes"]
         }
+        self.endpoints = {
```

Comment:
whats the difference between nodes and endpoints?

Reply:
We inject faults and run redpanda / kafka services on nodes, endpoints host kafkakv (key/value <-> kafka proxy). It may be the same or different machines.